src/utils/htmldoc.ts

```diff
@@ -0,0 +1,78 @@
+export const emailTemplate = (username: any) => `
```

Comment:
Please look into the use of `EJS` templating engine to handle the html rendering

[https://ejs.co/](https://ejs.co/)